### PR TITLE
Update the readme file for Leap's collection of examples

### PR DESCRIPTION
**Pull request by JoelPasvolsky**
Added content from the JN plus installation section

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,19 +1,60 @@
...
+create hybrid workflows, and develop custom hybrid components.
 
-Notebook 01 will 
-start you off with Leap's cloud-based hybrid solvers and out-of-the-box 
-dwave-hybrid samplers and workflows, and requires only familiarity with the binary 
-quadratic model (BQM) approach to formulating problems for solution on quantum 
-computers. 
+* **Notebook 01** will start you off with Leap's cloud-based hybrid solvers and
+  out-of-the-box dwave-hybrid samplers and workflows, and requires only familiarity
+  with the binary quadratic model (BQM) approach to formulating problems for
```
Comment (randomir) [inline README.md:8]: Perhaps link to BQM intro in Ocean docs?

File: README.md [hunk 2]
```diff
@@ -1,19 +1,60 @@
...
+solve problems, exploiting the complementary strengths that each provides. As
+quantum processors grow in size, offloading hard optimization problems to quantum
+computers promises performance benefits similar to CPUs' outsourcing of
+compute-intensive graphics-display processing to GPUs.
+
+This [Medium Article](https://medium.com/d-wave/three-truths-and-the-advent-of-hybrid-quantum-computing-1941ba46ff8c)
+provides an overview of, and motivation for, hybrid computing.
+
+D-Wave's [Leap quantum cloud service](https://cloud.dwavesys.com/leap) provides
+cloud-based hybrid solvers you can submit arbitrary BQMs to. These solvers, which
```
Comment (randomir) [inline README.md:27]: Not just BQMs anymore; but not sure if we want to mention that here since dwave-hybrid hasn't been quite updated for DQM/CQM.

**Review by randomir — APPROVED** [review]
LGTM

File: README.md (hunk 2 above)
Comment (JoelPasvolsky) [reply README.md]: changed to "quadratic models"

**Pull request merged**